osxcollector/output_filters/util/blacklist.py

```diff
@@ -64,6 +66,7 @@ def __init__(self, name, blacklisted_keys, file_path, is_regex=False, is_domains
                     self._blacklisted_values.append(line)
 
         self._blacklisted_values = [self._convert_to_matching_term(val) for val in self._blacklisted_values]
+        self._blacklisted_values = filter(None, self._blacklisted_values)
```

Comment:
I am curious if it is possible to have this filter somehow as a part of the list comprehension in the line before.